lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -2661,6 +2661,16 @@ public function getCurrentTimestampSQL()
     }
 
     /**
+     * Returns the SQL specifier for the platform to get a random number
+     *
+     * @return string
+     */
+    public function getRandomFunctionSQL()
+    {
+        return 'RAND()';
```

Comment:
I doubt this is SQL-92 standard. Maybe better to throw an exception here in the first place.